modules/gapi/test/infer/gapi_infer_onnx_test.cpp

```diff
@@ -237,6 +238,25 @@ void remapSSDPorts(const std::unordered_map<std::string, cv::Mat> &onnx,
     remapToIESSDOut({num_detections, detection_boxes, detection_scores, detection_classes}, ssd_output);
 }
 
+void remapRCNNPorts(const std::unordered_map<std::string, cv::Mat> &onnx,
+                          std::unordered_map<std::string, cv::Mat> &gapi) {
+    // Simple copy for outputs
+    const cv::Mat& in_boxes = onnx.at("6379");
+    const cv::Mat& in_scores = onnx.at("6383");
+    GAPI_Assert(in_boxes.depth()  == CV_32F);
+    GAPI_Assert(in_scores.depth() == CV_32F);
+    cv::Mat& out_boxes = gapi.at("out1");
+    cv::Mat& out_scores = gapi.at("out2");
+
+    copyToOut<float>(in_boxes, out_boxes);
+    copyToOut<float>(in_scores, out_scores);
+}
+
+void wrongRemapSSD(const std::unordered_map<std::string, cv::Mat> &/*onnx*/,
```

Comment:
alignment is broken.
BTW the name doesn't reflect the meaning ('reallocation`)

Reply:
Changed name. Fixed alignment.